src/main/java/seedu/address/logic/parser/CheckoutCommandParser.java

```diff
@@ -17,8 +22,24 @@
      * @throws ParseException if the user input does not conform the expected format
      */
     public CheckoutCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(args, PREFIX_ROOM, PREFIX_DATE_START, PREFIX_DATE_END);
+        if (!argMultimap.getValue(PREFIX_ROOM).isPresent()
```

Comment:
Should use `arePrefixesPresent`?

Reply:
No. `PREFIX_ROOM` is compulsory, but `PREFIX_START` and `PREFIX_END` are not. Similarly, if `PREFIX_START` is specified, `PREFIX_END` must also be specified. That is why we do not use `arePrefixesPresent` here.

Reply:
Yep! Can still be done using `arePrefixesPresent`. Would need multiple steps.

This method done here calls an unnecessary tokenisation, and is inconsistent with the parsing of other `Command`s.